protocol-designer/src/liquid-defs/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { getAllLiquidClassDefs } from '@opentrons/shared-data'
+
+const liquidClassDefs = getAllLiquidClassDefs()
+export const getLiquidClassDisplayName = (
+  liquidClass: string | null
+): string | null => {
+  if (liquidClass == null) {
+    return null
+  }
+  return liquidClassDefs[liquidClass]?.displayName ?? null
```

Comment:
nit, would be good to do a console.warn if liquidClassDefs[liquidClass]?.displayName is not found for debugging purposes

```suggestion
  if (liquidClass == null) {
    return null
  }
  
  if (liquidClassDefs[liquidClass] == null) {
    console.warn('oops')
  }
  return liquidClassDefs[liquidClass]?.displayName ?? null
```

Reply:
This is a good call